### PR TITLE
Tickets/dm 30283

### PR DESCRIPTION
**Pull request by frossie**
PR for management approval.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,58 +1,129 @@
...
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+   The process is complete when the relevant AD(s) and/or Director reviews the note and the note is published (merged to master).
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+.. Add content here.
 
-     Caption text.
+What is covered
```
Comment (womullan) [inline index.rst]: Perhaps as a  formal document we should use more formal terms .. "Scope" for example here. 
```suggestion
Scope 
```

File: index.rst [hunk 2]
```diff
@@ -1,58 +1,129 @@
...
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+This delivery note covers the Discourse-based web forum service deployed as community.lsst.org
 
-   Feel free to delete this instructional comment.
+The web forum is a destination for long-lived (compared to Slack) discussion forum replacing mailing lists for internal project use, as well as a community-facing forum designed to involve the community in discussions with the project as well as their peers in an asynchronous and searchable manner.
 
-:tocdepth: 1
+What artifacts (code, docs, infrastructure) are involved?
```
Comment (womullan) [inline index.rst]: Again on the formality not sure phrasing each heading as a question ticks the formal boxes.


```suggestion
Artifacts (code, docs, infrastructure) involved in this delivery
```

File: index.rst [hunk 3]
```diff
@@ -1,58 +1,129 @@
...
+- Let's Encrypt account for TLS
+- AWS S3 bucket for backups
+- Slack application to bridge Discourse and Slack
+- Twitter app for OAuth login
+- GitHub OAuth app for login
+- 1Password for secrets management
+
+The motivation for the service's creation and outline of its elements can be found at https://sqr-011.lsst.io/#community-forum-and-mailing-lists
+
+What costs does the service incur?
```
Comment (womullan) [inline index.rst]:
```suggestion
Potential costs
```

File: index.rst [hunk 4]
```diff
@@ -1,58 +1,129 @@
...
+-----------------------------------
+
+The service is hosted on a Digital Ocean droplet and costs approximately $30/month billed directly to an AURA Purchasing credit card under a blanket PO against CS008.1.02C.10.01.KLM21001A
+
+How is the service supported?
```
Comment (womullan) [inline index.rst]:
```suggestion
Ongoing support
```

File: index.rst [hunk 5]
```diff
@@ -1,58 +1,129 @@
...
+
+The slack channel #rubinobs-forum-team offers support of and discussion with content administators of the forum, such as the Ops Community Engagement Team.
+
+Users of the forum can discuss management issues of the forum on the forum itself under the Meta category: https://community.lsst.org/c/meta/3
+
+
+Handover
+========
+
+What construction team is delivering the service?
```
Comment (womullan) [inline index.rst]:
```suggestion
Delivering team
```

File: index.rst [hunk 6]
```diff
@@ -1,58 +1,129 @@
...
+-------------------------------------------------
+
+SQuaRE built and operates this service during Construction on behalf of Data Management.
+
+As a project forum, content was contributed organically primarily by project staff.
+
+What operations team is receiving the service?
```
Comment (womullan) [inline index.rst]: Receiving operations team

File: index.rst [hunk 7]
```diff
@@ -1,58 +1,129 @@
...
+
+SQuaRE will continue operating the service during Operations on behalf of Data Production.
+This includes all technical aspects of operating the service including its deployment, technical and design upgrades as well as security updates, backups, etc.
+
+The Community Engagement Team will be responsible for content management of the sections of the forum accessible to the Rubin Scientific Community on behalf of Rubin System Performance.
+It is also responsible for setting policy in areas that are relevant to its activities, eg. what the protocol is for answering user questions.
+
+The teams use the Slack channel #rubinobs-forum-team to communicate intended work, discuss improvements and genereally keep the other informed.
+
+Is there any authorisation or asset tranfer required?
```
Comment (womullan) [inline index.rst]:
```suggestion
Authorization or asset transfer
```

**Review by womullan — DISMISSED** [review]
formal tone

File: index.rst [hunk 8]
```diff
@@ -1,58 +1,129 @@
...
+- Twitter app for OAuth login
+- GitHub OAuth app for login
+- 1Password for secrets management
+
+The motivation for the service's creation and outline of its elements can be found at https://sqr-011.lsst.io/#community-forum-and-mailing-lists
+
+What costs does the service incur?
+-----------------------------------
+
+The service is hosted on a Digital Ocean droplet and costs approximately $30/month billed directly to an AURA Purchasing credit card under a blanket PO against CS008.1.02C.10.01.KLM21001A
```
Comment (rblum5) [inline index.rst]: Suggest "The service is hosted on a Digital Ocean droplet and costs approximately $30/month billed directly to an AURA Purchasing credit card under a blanket PO against CS008.1.02C.10.01.KLM21001A. A new PO will be issued for Operations under CS002."

**Review by rblum5 — COMMENTED** [review]
This all looks great. I put in a suggestion in sec 1.2

File: index.rst (hunk 8 above)
Comment (frossie) [reply index.rst]: Thanks, the billing for this is on the same invoice as the mailing list server (lists.lsst.org) but I will (1) turn around a quick delivery note for that (2) it's only $10/month

**Pull request merged**